src/t2iapi/metric/service.proto

```diff
@@ -35,10 +35,12 @@ service MetricService {
         returns (BasicResponse);
 
     /*
-    Change the value for the metric with the provided handle and confirm the value using the device ui.
-     */
-    rpc SetUserConfirmableValue (BasicHandleRequest)
-        returns (BasicResponse);
+    For the metric with the given handle do the following simultaneously:
+     - set the @ActivationState to the given value
+     - set the pm:MetricValue/@Value resp. @Samples and confirm the value resp. samples using the device ui
+   */
+   rpc SetActivationStateAndUserConfirmableValue (t2iapi.metric.SetActivationStateRequest)
```

Comment:
i would suggest renaming to SetActivationStateAndUserConfirmableValueRequest to mitigate confusion

Reply:
done